gdpr/impl.go

```diff
@@ -241,3 +246,22 @@ func (a AlwaysAllow) BidderSyncAllowed(ctx context.Context, bidder openrtb_ext.B
 func (a AlwaysAllow) AuctionActivitiesAllowed(ctx context.Context, bidder openrtb_ext.BidderName, PublisherID string, gdprSignal Signal, consent string, weakVendorEnforcement bool) (allowBidRequest bool, passGeo bool, passID bool, err error) {
 	return true, true, true, nil
 }
+
+// Mock Vendor claims everything.
```

Comment:
I wouldn't call this a mock, that terminology is usually reserved for tests. Consider something like: `// vendorTrue claims everything`